std/experimental/allocator/package.d

```diff
@@ -605,6 +605,8 @@ unittest
     assert(a == [0, 0, 0]);
 }
 
+import std.experimental.allocator.gc_allocator: isGCSafeAllocator;
```

Comment:
> I just read the entire history of 314 - pretty scary :fearful: 

Yeah, it was classified as a non-issue by Andrei and Walter (which IMO was a mistake) which is why it went unfixed for so long

> so I basically should do import std.experimental.allocator.gc_allocator, because that doesn't leak?

Yes, and patiently wait for the day when it's actually fixed and not a deprecation